src/objective-c/tests/BUILD

```diff
@@ -24,13 +24,22 @@ load("@build_bazel_rules_apple//apple:resources.bzl", "apple_resource_bundle")
 load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application", "ios_unit_test")
 load("@build_bazel_rules_apple//apple:macos.bzl", "macos_unit_test")
 load("@build_bazel_rules_apple//apple:tvos.bzl", "tvos_application", "tvos_unit_test")
+load("@build_bazel_rules_apple//apple/testing/default_runner:ios_test_runner.bzl", "ios_test_runner")
 
 licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
 exports_files(["LICENSE"])
 
+ios_test_runner(
```

Comment:
note that in bazel we already have this mechanism:
https://github.com/grpc/grpc/blob/adfd009d3a255b825ea91959620c11805418b22b/src/objective-c/tests/BUILD#L60
which is what we are currently using (and the bazel test are passing). I think your approach is better and more general, but we probably shouldn't have two different mechanisms for achieving the same thing.